thorlcr/graph/thgraph.hpp

```diff
@@ -1010,6 +1019,16 @@ class graph_decl CJobChannel : public CInterface, implements IGraphCallback, imp
 
 interface IOutputMetaData;
 
+inline activity_id createCompoundActSeqId(activity_id actId, byte seq)
+{
+    if (seq)
+    {
+        dbgassertex(actId <= 0xffff); // normal max is MAX_ACTIVITY_ID (0xffffff), reserving 0x00ff0000 byte for seq if present
```

Comment:
This limit is too low.  I have an example (issue16893) with over 200,000 activities in a graph.